basics/settings.md

```diff
@@ -3,6 +3,10 @@ title: Settings
 ---
 <!-- Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
 
-* [Persisting State of Components](/basics/persisting_state_of_components.md)
-* [Persisting Sensitive Data](/basics/persisting_sensitive_data.md)
-* Editing Settings
+## Introduction to Settings
+Settings are but one application of the IntelliJ Platform [Persistence Model](/basics/persistence.md).
+For more information, see:
```

Comment:
link to https://www.jetbrains.com/help/idea/configuring-project-and-ide-settings.html